rfc/src/rfcs/0009-function-contracts.md

```diff
@@ -37,26 +39,28 @@ verification, which paves the way for the following two ambitious goals.
     and potential remedies can be found in the [future](#future-possibilities)
     section.
 
-- **Scalability:** Function contracts are sound (over)abstractions of function
-  behavior. By verifiying the contract against its implemetation and
-  subsequently performing caller verification against the (cheaper) abstraction,
-  verification can be modularized, cached and thus scaled.
+- **Scalability:** A function contract is an abstraction (sound
+  overapproximation) of a function's behavior. By verifying the contract against
+  its implementation and subsequently performing caller verification against the
+  (cheaper) abstraction, verification can be modularized, cached and thus
+  scaled.
 - **Unbounded Verification:** The abstraction provided by the contract can be
   used instead of a recursive call, thus allowing verification of recursive
   functions.
 
 Function contracts are completely optional with no user impact if unused. The
-newly introduced APIs are entirely backwards compatible.
+newly introduced interface (attributes, macros, CLI) do not require any changes
```

Comment:
Sorry, but that last sentence still doesn't make sense. I still think the first sentence is sufficient to say that this RFC is backward compatible. If you really want to reinforce the message, I would say something like:

> Function contracts are completely optional with no user impact if unused. This RFC proposes the addition of new attributes, and functions, that shouldn't interfere with existing functionalities. 

You could also add:

> Adding contract to a function will not change how this function is used in existing harnesses.